src/main/java/net/mcreator/blockly/data/BlocklyLoader.java

```diff
@@ -26,41 +33,67 @@ public static void init() {
 		INSTANCE = new BlocklyLoader();
 	}
 
-	private final ExternalBlockLoader procedureBlockLoader;
-	private final ExternalBlockLoader jsonTriggerLoader;
-	private final ExternalBlockLoader aitaskBlockLoader;
-	private final ExternalBlockLoader cmdargsBlockLoader;
-	private final ExternalBlockLoader featureBlockLoader;
+	private static final List<String> builtinCategories = new ArrayList<>() {{
+		add("other");
+		add("apis");
+		add("mcelements");
+		add("mcvariables");
+		add("customvariables");
+		add("logicloops");
+		add("logicoperations");
+		add("math");
+		add("text");
+		add("time");
+		add("advanced");
+		add("action");

```

Comment:
This was called actions before, not action. Are you sure this is right?

Reply:
I surely removed the `s` by accident and I never noticed it.